lib/worker/src/main.rs

```diff
@@ -87,7 +91,7 @@ enum EvmType {
     Remote,
 }
 
-#[tokio::main(flavor = "current_thread")]
+#[tokio::main(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
`w3s-rs` lib needs multiple threaded runtime because when the files are larger than 100MB it splits them into multiple car file and uploads them concurrently. it blocks the current thread to wait for all the concurrent jobs to finish their uploads.